server/src/main/java/io/crate/exceptions/GroupByOnArrayUnsupportedException.java

```diff
@@ -32,7 +32,7 @@
 public class GroupByOnArrayUnsupportedException extends ElasticsearchException implements UnscopedException {
 
     public GroupByOnArrayUnsupportedException(String columnName) {
-        super("Column \"" + columnName + "\" has a value that is an array. Group by doesn't work on Arrays");
+        super("Column \"" + columnName + "\" has a value that is an array. Group by doc values doesn't work on Arrays");
```

Comment:
Not sure, code should not enter this section - (docvalues are not supported for arrays and we throw it from there) - just added a small hint where to look for if we see it